analyzers/md5hashanalyzer.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+
+class MD5HashAnalyzer(RegexAnalyzer):
+    """Analyzer to match MD5 password hashes via regex"""
+    name = "MD5HashAnalyzer"
+
+    def __init__(self, action):
+        regex = "[a-f0-9]{32}"
```

Comment:
I just thought about the issue which might come up from hashes... Normal text might also match. Or b64 encoded data. But that's hard to "fix" since there is not necessarily a newline or something like that in the end of each hash.

I like the analyzers anyway.

Reply:
yeah, these will definitely generate some false positives occasionally. they also will not account for  abbreviated hashes like git likes to use for commits, for example. So it's far from perfect, but it's a start.